misc/toml-parser/src/main/java/io/ballerina/toml/internal/parser/TomlParserErrorHandler.java

```diff
@@ -221,6 +232,7 @@ protected Result seekMatch(ParserRuleContext currentCtx, int lookahead, int curr
                     return seekInAlternativesPaths(lookahead, currentDepth, matchingRulesCount, alternativeRules,
                             isEntryPoint);
                 case VALUE:
+                case INLINE_TABLE_ENTRY_START:
                     alternativeRules = VALUE;
```

Comment:
`INLINE_TABLE_ENTRY_START` is not a value only, it has to be a key-value pair?
So the next rule has to be `ParserRuleContext.KEY_VALUE_PAIR`?